libs/partners/snowflake/docs/chat.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
Is it possible to move this file to `docs/docs/integrations/chat/` folder where all chat model examples live?